docs/troubleshooting/harvester.md

```diff
@@ -4,7 +4,53 @@ sidebar_label: Harvester
 title: ""
 ---
 
-## Generate a support bundle
+## Fail to Deploy a Multi-node Cluster due to Incorrect HTTP Proxy Setting
+
+### ISO Installation Without a Harvester Configuration File
+
+#### Configure HTTP Proxy During Harvester Installation
+
+In some environments, you configure [http-proxy](../airgap/#configure-an-http-proxy-during-installation) of [OS Environment](../install/harvester-configuration.md#osenvironment) during Harvester installation.
+
+#### Configure HTTP Proxy After First Node is Ready
+
+After the first node is installed successfully, you login into the `Harvester GUI` to configure [http-proxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) of [Harvester System Settings](../install/harvester-configuration.md#system_settings).
+
+Then you continue to add more nodes into the cluster.
+
+#### One Node Becomes Unavailable
+
+The issue you may encounter:
+
+```
+The first node is installed successfully.
+
+The second node is installed successfully.
+
+The third node is installed successfully.
+
+Then the second node changes to Unavialable state and cannot recover automatically.
+```
+
+#### Solution
+
+When the nodes in the cluster do not use the HTTP Proxy to communicate with each other, after the first node is installed successfully, you need to configure [http-proxy.noProxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) against the CIDR used by those nodes.
+
+For example, your cluster assigns IPs from CIDR `172.26.50.128/27` to nodes via DHCP/static setting, please add this CIDR to `noProxy`.
```

Comment:
```suggestion
For example, your cluster assigns IPs from CIDR `172.26.50.128/27` to nodes via DHCP/static setting. This CIDR must then be added to `noProxy`.
```

Reply:
I am not sure if it is goog to break this `for example` into two sentences, it explians one thing.